netlify.toml

```diff
@@ -2,15 +2,16 @@
   # Directory (relative to root of your repo) that contains the deploy-ready
   # HTML files and assets generated by the build. If a base directory has
   # been specified, include it in the publish directory path.
-  publish = "docs/export/"
+  publish = "export/"
+  base = "docs/"
```

Comment:
base directory must be the one that contains dependency management according to https://docs.netlify.com/configure-builds/monorepos/#yarn-workspaces

Reply:
It's not clear if they mean root dependencies or site's dependencies. After all, to build a site we must install dependencies for the whole monorepo.

Reply:
Dependency management files according to netlify contain `.nvmrc` and `package.json`, they don't explicitly mention it, but I don't see how `pnpm-lock.yaml` is not a dependency management file. It's indeed a bit unclear though.

Reply:
Yeah, there is some uncertainty here, but given the [Recommended monorepo setup](https://docs.netlify.com/configure-builds/monorepos/#recommended-monorepo-setup) I think my initial assumption was misplaced. 🙈